src/main/java/mate/academy/dto/category/CreateCategoryRequestDto.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.dto.category;
+
+import jakarta.validation.constraints.NotEmpty;
+
+public record CreateCategoryRequestDto(
+        @NotEmpty
```

Comment:
```suggestion
        @NotBlank
```

You can also validate size for both fields